contents/handbook/growth/marketing/events-partnerships.mdx

```diff
@@ -0,0 +1,107 @@
+- **Is the event itself interesting?** Check the event’s purpose, principles, and participants before agreeing. We don’t do boring partnerships or events. We focus on makers. We like new things. 
+- **Can PostHog help bring it to the next level?**  Ideally, our role is to make an event “more interesting,” not more of the same. This can include practical technology components and demos, an interesting branded experience or collab.
+- **Is there a reasonable time window for collaborating?** Aim for a minimum of 8 weeks out. We find that 4 weeks out is the latest we can get involved in an event, because of the need to arrange physical goods, travel, attendance and promotion. 
+
+### Checklist of questions for sponsoring events: 
+Ask these questions to determine what specific tasks PostHog might need to do: 
+- **People:** Are PostHog people needed in person? What level, role or skill? 
+- **Asks of the people:** Will there be a talk? Networking? Demos? Participation?
+- **Goods/merch:** Will physical objects (shirts, giveaways, signage) be needed?
+- **Social media:** What kind of co-promoting is expected?
+- **Artwork and branding:** Are unique, custom assets needed?
+- **Online support:** Will live and realtime support or communication be needed during the event?
+- **Communication:** In-house at PostHog, and beyond, what needs to be shared? 
+- **Travel:** Is travel going to be part of it? Who, when, where, how? 
+- **Timeline:** When is it happening, and when do we need all this by?
+
+### Make your plan and build your team:
+- Based on the above checklist, pick an event owner, and if needed, a temporary small team. This team dissolves after the event, but takes seriously the work of delivering well. 
+- Create an Issue that covers the event sponsorship, and assign it to the event owner. [Example event Issue](https://github.com/PostHog/company-internal/issues/1759)
+
+### Showing up to a partner event sponsorship as PostHog:
+- Travel in such a way that you can arrive on-time, well-rested, early and ready to support well
+- Plan in advance to bring everything you need: laptop, adapters, slides, merch, signs, etc.
+- Have speaking remarks prepared, and anticipate playing a role speaking on-stage to give PostHog further visibility 
+- Wear PostHog merch so you are visible part of the team
+- Act as a co-host, supporting the hosting team, and offering encouragement as the event goes along
+- While business cards are perhaps antiquated, have a plan for sharing contact info with people you meet (LinkedIn, Bump, etc)
+- Be on a mission to meet people; it’s part of the event function. Ask and answer questions. 
+- Don’t cluster in a clique: If attending with our PostHog teammates, spread out, don’t loiter as a wolfpack
+- Finish line: know what constitutes “done” for your presence at the event (don’t leave early or without checking with the host)
+
+## Planning and running a PostHog branded event:
```

Comment:
We can probably just cut this entire section. We have no plans to run our own events regularly and the events we do occasionally run (e.g. AI startup school idea) are so bespoke that the guidance here won't be all that useful.